model/flow/identifier.go

```diff
@@ -188,3 +191,26 @@ func Sample(size uint, ids ...Identifier) []Identifier {
 	}
 	return dup[:size]
 }
+
+func CidToFlowID(c cid.Cid) (Identifier, error) {
+	decoded, err := mh.Decode(c.Hash())
+
+	if err != nil {
+		return ZeroID, fmt.Errorf("failed to decode CID: %w", err)
+	}
+
+	if decoded.Code != mh.SHA2_256 {
+		return ZeroID, fmt.Errorf("unsupported CID hash function: %v", decoded.Name)
+	}
+
+	if decoded.Length != IdentifierLen {
+		return ZeroID, fmt.Errorf("invalid CID length: %d", decoded.Length)
+	}
+
+	return HashToID(decoded.Digest), nil
+}
+
+func FlowIDToCid(f Identifier) cid.Cid {
```

Comment:
any easy unit tests for these two new functions?

Reply:
Sure, will add.